backend/workflow_manager/endpoint/destination.py

```diff
@@ -528,17 +528,20 @@ def _push_to_queue(
             settings=connector_settings, connector_id=connector.connector_id
         )
         with source_fs.open(input_file_path, "rb") as remote_file:
+            whisper_hash = None
             file_content = remote_file.read()
             # Convert file content to a base64 encoded string
             file_content_base64 = base64.b64encode(file_content).decode("utf-8")
             q_name = f"review_queue_{self.organization_id}_{workflow.id}"
+            if meta_data:
+                whisper_hash = meta_data.get("whisper-hash")
```

Comment:
In this particular case I think we can initialise whisper_hash using else clause rather than doing it separately. 
```suggestion
            if meta_data:
                whisper_hash = meta_data.get("whisper-hash")
            else:
                whisper_hash = None
```